x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group1/disable.ts

```diff
@@ -172,6 +185,54 @@ export default function createDisableRuleTests({ getService }: FtrProviderContex
       });
     });
 
+    it('should not untrack alerts if untrack is false', async () => {
```

Comment:
Question: Do you think we should update this test `should create untracked-instance events for all alerts` to match your technique I like better to query the alert index than the kibana event log. What do you think?